pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/security/ApexCRUDViolationRule.java

```diff
@@ -337,6 +339,19 @@ private boolean isLastMethodName(final ASTMethodCallExpression methodNode, final
         return false;
     }
 
+    private boolean isWithSecurityEnforced(final AbstractApexNode<?> node) {
+//        if (node instanceof ASTSoqlExpression) {
+//            String pattern = "(?i).*[^']\\s*WITH SECURITY_ENFORCED\\s*[^']*";
+//            String query = ((ASTSoqlExpression) node).getQuery();
+//            return query.matches(pattern);
+//        }
+//        return false;
+          if (node instanceof ASTSoqlExpression) {
+            return WITH_SECURITY_ENFORCED.matcher(((ASTSoqlExpression) node).getQuery()).matches();
+          }
+          return false;
+    }
```

Comment:
There seems to be some indentation problem which is failing the build. It seems, these lines are indented with two spaces too much...

```
[ERROR] src/main/java/net/sourceforge/pmd/lang/apex/rule/security/ApexCRUDViolationRule.java:[349] (indentation) Indentation: 'if' has incorrect indentation level 10, expected level should be 8.
[ERROR] src/main/java/net/sourceforge/pmd/lang/apex/rule/security/ApexCRUDViolationRule.java:[351] (indentation) Indentation: 'if rcurly' has incorrect indentation level 10, expected level should be 8.
[ERROR] src/main/java/net/sourceforge/pmd/lang/apex/rule/security/ApexCRUDViolationRule.java:[352] (indentation) Indentation: 'method def' child has incorrect indentation level 10, expected level should be 8.
```